icons/square-function.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 4a2 2 0 00-2-2H4a2 2 0 00-2 2v16a2 2 0 002 2h16a2 2 0 002-2V4z" />
+  <path d="M9 11.237h2.848M9 18c1.934 0 2.848-.996 2.848-2.793v-3.97m0 0h2.813m-2.813 0V8.984c0-1.937 1.034-3.265 3.152-2.933" />
```

Comment:
```suggestion
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
```

Reply:
@ericfennis Why `width` and `height` in `rect` tag?

Reply:
`width` & `height` are used by the rect svg element to define the width and height 😛 .  See [docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/rect)